components/app-framework/src/widgets/detail-view/detail-view-dialog.scss

```diff
@@ -29,6 +31,10 @@
     width: $console-input-width * 1.25 + 2 * $console-unit-space;
   }
 
+  &.dialog-form-wizard > .modal-dialog .detail-view-content {
+    width: $console-input-width * 3 + 2 * $console-unit-space;
```

Comment:
Only those that contain the dialog-form-wizard class, which at the moment is only the new modal